recipe/meta.yaml

```diff
@@ -1,58 +1,72 @@
 {% set name = "aws-parallelcluster" %}
-{% set version = "2.11.9" %}
+{% set version = "3.6.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 615de4d59d9fd56a31d4feb3aeefe685346538a8dd0c1c35b660029f891d4dfd
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aws-parallelcluster-{{ version }}.tar.gz
```

Comment:
This is the new template for our bot. If I recall correctly PyPI change how they handle case-sensitive names and it is safer to use the proper name instead of the jinja variable. Anyway, I'd leave this to our bot as it will always be updated with latest recipe template.